bin/build_couchdb_lucene.js

```diff
@@ -0,0 +1,149 @@
+/*
```

Comment:
We could do with some comments explaining the purpose and strategy of this file - together with a few impl comments